pkg/codegen/python/gen.go

```diff
@@ -827,7 +832,8 @@ func (mod *modContext) genResource(res *schema.Resource) (string, error) {
 	fmt.Fprintf(w, "            '%s',\n", tok)
 	fmt.Fprintf(w, "            resource_name,\n")
 	fmt.Fprintf(w, "            __props__,\n")
-	fmt.Fprintf(w, "            opts)\n")
+	fmt.Fprintf(w, "            opts,\n")
+	fmt.Fprintf(w, "            remote=True)\n")
```

Comment:
We don't always want to emit `remote=True`. Should this only be set when `res.IsComponent` is true?

Reply:
🤦  Excellent point.

Reply:
Edited my comment ☝️

Reply:
Hmm...I guess `IsComponent` shouldn't necessarily imply a remote component. I think we can infer the remote status from the token though, since it should include an additional prefix (e.g., `/aws/v3.1.0/schema.json`)

Reply:
So probably we need a remote property in the bound structs, but not in the JSON.

Does that sound right @pgavlin ?

Reply:
Confirmed offline that `ComponentResource` classes should always set `remote=True`